python/private/python_repositories.bzl

```diff
@@ -54,6 +55,15 @@ def py_repositories():
         internal_config_repo,
         name = "rules_python_internal",
     )
+    maybe(
+        hub_repo,
+        name = "pythons_hub",
```

Comment:
Hrm. How do we get the list of values into pythons_hub for workspace builds?
Or is the idea: we just want the repo to _exist_ so that load() can work?